_episodes/|1-06-issues.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Using GitHub Issues"
+teaching: 15
+exercises: 10
+questions:
+- "How can I use GitHub issues to manage a TODO list for my project?"
+objectives:
+- "Explain what GitHub issues are and how to create them."
+- "Explain how issues can help you keep track of your work in GitHub"
+- "How to claim issues."
+- "Learn how to use GitHub functionality to effectively communicate who is working on what."
+keypoints:
+- "Issues are a feature of GitHub which let you track work in a respository."
+- "GitHub provides functionality for referencing issues in comments"
+- "Task lists can be created to keep track of a list of issues"
+- "Formatting syntaxes, templates, and subscribing to issues help with communication efficiency"
+---
+
+## GitHub issues
+
+- GitHub issues are a feature of GitHub. They let you keep track of your work on a GitHub repository and are used to report bugs, request features or enhancements, or to discuss implementation details of some parts of the code.
+ - Issues are a kind of TODO list, with pending and completed tasks as well as serving to prioritise the development activity.
+ - Labels can be added to the issues by the repository administrator to inform at a first glance what the issue is about. Typical labels are “bug”, “enhancement”, “low priority” or “good first issue”, for example.
+ - Issues also can have one or more people assigned to them who will take care of sorting them out and closing them when done or if no longer relevant.
+ - By default any GitHub user can create an issue in a public repository
+
+The following figure shows some of the issues open in a certain repository. The labels tell us there are a couple of bug reports, a couple of issues related to the performance of the software and several ones that are simple enough to be tackled by novice people. Most of them have some discussion going on.
+
+ ![Git collaborative]({{ site.baseurl }}/fig/issues1.png "Example of issues."){:class="img -responsive"}
+
+## Creating GitHub issues
+It is possible to create issues in a number of ways (see the GitHub docs page on [creating issues][github-creating-issues] to learn about the various methods). One of the most common and simplist methods is to create an issue from a repository. This can be done by navigating to the main page of the repository on GitHub and click on `Issues`, then click on the `New Issue` button.
```

Comment:
```suggestion
It is possible to create issues in a number of ways (see the GitHub docs page on [creating issues][github-creating-issues] to learn about the various methods). One of the most common and simplest methods is to create an issue from a repository. This can be done by navigating to the main page of the repository on GitHub and click on `Issues`, then click on the `New Issue` button.
```